app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,9 @@
+class PostsController < ApplicationController
+  def index
+    render template: 'layouts/posts/index'
+  end
+
+  def show
+    render template: 'layouts/posts/show'
+  end
+end
```

Comment:
- [ ] You did a good job working on this milestone! 💯 However, I suggest that you follow the Rails' conventions for views:

Rails, by default, will look for views in `app/views/posts` in the case of the `PostController`, but you currently have them nested in your `layouts` folder. I think you should move them outside, so instead of specifying which template to render, Rails chooses it automagically! 🤓 🔧